spec/index.html

```diff
@@ -2036,7 +2036,7 @@ <h3>Algorithm</h3>
                   recursively processed by this algorithm.</li>
                 <li id="hndq-5-4-4">For each <var>related</var> in <var>p</var>:
                   <ol>
-                    <li id="hndq-5-4-4-1">If a canonical identifier has been issued for
+                    <li id="hndq-5-4-4-1">If an identifier has been issued for
                       <var>related</var> by <var>issuer</var>, append the string <code>_:</code>, followed by 
                       <var>related</var>, to <var>path</var>.</li>
```

Comment:
```suggestion
                      <var>related</var>, to <var>path</var>.
                      <details><summary>Explanation</summary>
                        <p>A canonical identifier may have been generated before calling this algorithm,
                            if it was issued from an earlier call to <a href="#hash-1d-quads">Hash First Degree Quads algorithm</a>.</p>
                      </details>
                    </li>

Reply:
> can you suggest an Explanation/details that we would add to 5.4.4.1 to clarify why this is so?

I think that suggested explanation is fine. If you want any other details: there's no reason to recurse and apply the algorithm to any related blank node that has already been assigned a canonical identifier. Furthermore, using the canonical identifier also further distinguishes it from any temporary identifier, allowing for even greater efficiency in finding the chosen path.